pkg/pods/manifest.go

```diff
@@ -131,10 +237,13 @@ func (manifest *Manifest) Write(out io.Writer) error {
 	return nil
 }
 
-// OriginalBytes() always returns the bytes from the original manifest file
-// that was passed to ManifestFromBytes(). Any mutations to the struct will not
-// appear in the returned bytes. If mutations are desired, use Marshal()
-func (manifest *Manifest) OriginalBytes() ([]byte, error) {
+// Returns bytes correlating to the (potentially mutated) struct, unlike
+// Bytes() which guarantees that the bytes returned will be the same bytes the
+// manifest struct was built from
```

Comment:
If we delete `manifest.raw` when getting a builder as well as the signature data, then you don't even need a different method `Marshal` and `Bytes`. Effectively can just have the behavior of `Marshal` be sufficient.